package.json

```diff
@@ -43,58 +43,58 @@
     "release:major": "npm run build && npm run release -- --release-as major"
   },
   "dependencies": {
-    "@angular/animations": "~8.2.0",
-    "@angular/common": "~8.2.0",
-    "@angular/compiler": "~8.2.0",
-    "@angular/core": "~8.2.0",
-    "@angular/forms": "~8.2.0",
-    "@angular/platform-browser": "~8.2.0",
-    "@angular/platform-browser-dynamic": "~8.2.0",
-    "@angular/router": "~8.2.0",
-    "@ngxs/logger-plugin": "~3.6.2-dev.master-14ab2bf",
-    "@ngxs/store": "~3.6.2-dev.master-14ab2bf",
+    "@angular/animations": "~9.1.9",
+    "@angular/common": "~9.1.9",
+    "@angular/compiler": "~9.1.9",
+    "@angular/core": "~9.1.9",
+    "@angular/forms": "~9.1.9",
+    "@angular/platform-browser": "~9.1.9",
+    "@angular/platform-browser-dynamic": "~9.1.9",
+    "@angular/router": "~9.1.9",
+    "@ngxs/logger-plugin": "~3.6.2",
+    "@ngxs/store": "~3.6.2",
     "reflect-metadata": "~0.1.13",
-    "rxjs": "~6.4.0",
-    "tslib": "~1.10.0",
-    "zone.js": "~0.9.1"
+    "rxjs": "~6.5.5",
+    "tslib": "~2.0.0",
+    "zone.js": "~0.10.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.802.0",
-    "@angular-devkit/build-ng-packagr": "~0.802.2",
-    "@angular/cli": "~8.2.0",
-    "@angular/compiler-cli": "~8.2.0",
-    "@angular/language-service": "~8.2.0",
-    "@types/jasmine": "~3.3.8",
-    "@types/jasminewd2": "~2.0.3",
-    "@types/node": "~8.9.4",
-    "@typescript-eslint/eslint-plugin": "^2.28.0",
-    "@typescript-eslint/eslint-plugin-tslint": "^2.28.0",
-    "@typescript-eslint/parser": "^2.28.0",
-    "codelyzer": "~5.0.0",
-    "commitizen": "^4.0.4",
-    "conventional-changelog-cli": "^2.0.31",
-    "coveralls": "^3.0.13",
-    "cz-conventional-changelog": "^3.1.0",
-    "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.10.1",
+    "@angular-devkit/build-angular": "~0.901.7",
+    "@angular-devkit/build-ng-packagr": "~0.901.7",
+    "@angular/cli": "~9.1.7",
+    "@angular/compiler-cli": "~9.1.9",
+    "@angular/language-service": "~9.1.9",
+    "@types/jasmine": "~3.5.10",
+    "@types/jasminewd2": "~2.0.8",
+    "@types/node": "~14.0.9",
+    "@typescript-eslint/eslint-plugin": "^3.1.0",
+    "@typescript-eslint/eslint-plugin-tslint": "^3.1.0",
+    "@typescript-eslint/parser": "^3.1.0",
+    "codelyzer": "~5.2.2",
+    "commitizen": "^4.1.2",
+    "conventional-changelog-cli": "^2.0.34",
+    "coveralls": "^3.1.0",
+    "cz-conventional-changelog": "^3.2.0",
+    "eslint": "^7.1.0",
+    "eslint-config-prettier": "^6.11.0",
```

Comment:
I think we should avoid updating dependencies that don't relate to the actual issue

Reply:
Yes I'm good with that.  I am personally pretty aggressive about updating packages but I think you are right to play it safe.